### PR TITLE
[Relay][Frontend][TFlite] Add parses support for unary elemwise ops

### PR DESCRIPTION
**Pull request by inadob**
* Add generic method to convert unary math functions
* Add support for: abs, exp, ceil, floor, log, sin, cos, sqrt, rsqrt, neg
* Add relevant tests

### CODE REVIEW

**Comment by inadob** [comment]
@kevinthesun @FrozenGene can you please review this patch

File: tests/python/frontend/tflite/test_forward.py [hunk 1]
```diff
@@ -616,6 +616,109 @@ def test_forward_concatenation():
...
+#######################################################################
+# Floor
+# -----
+
+def _test_floor(data):
+    """ One iteration of floor """
+    return _test_unary_elemwise(math_ops.floor, data)
+#######################################################################
+# Exp
+# -----
```
Comment (anijain2305) [inline tests/python/frontend/tflite/test_forward.py]: 2 extra "--".

**Comment by tqchen** [comment]
ping @FrozenGene please also take a look

File: tests/python/frontend/tflite/test_forward.py [hunk 2]
```diff
@@ -616,6 +616,109 @@ def test_forward_concatenation():
...
+#######################################################################
+# Neg
+# ---
+
+def _test_neg(data):
+    """ One iteration of neg """
+    return _test_unary_elemwise(math_ops.neg, data)
+#######################################################################
+
+def _test_forward_unary_elemwise(testop):
```
Comment (FrozenGene) [inline tests/python/frontend/tflite/test_forward.py]: testop -> test_op

Comment (inadob) [reply tests/python/frontend/tflite/test_forward.py]: That's how the binary elementwise operators were implemented. Do you want me to change both?

Comment (FrozenGene) [reply tests/python/frontend/tflite/test_forward.py]: Yes. I think you should only need to change this too:
```
    if test_op in {'_test_log', '_test_sqrt', '_test_rsqrt'}:
        test_op(np.arange(6.0, dtype=np.float32).reshape((2, 1, 3)))
        test_op(np.arange(6.0, dtype=np.int32).reshape((2, 1, 3)))
```

Comment (FrozenGene) [reply tests/python/frontend/tflite/test_forward.py]: Yes. I think you should only need to change this too:

    if test_op in {'_test_log', '_test_sqrt', '_test_rsqrt'}:
        test_op(np.arange(6.0, dtype=np.float32).reshape((2, 1, 3)))
        test_op(np.arange(6.0, dtype=np.int32).reshape((2, 1, 3)))

**Comment by tqchen** [comment]
Thanks @FrozenGene @anijain2305 @inadob

**Pull request merged**